glTFLoader/glTFLoader.csproj

```diff
@@ -57,27 +57,28 @@
     <Compile Include="Schema\Channel.cs" />
     <Compile Include="Schema\ChildOfAGltfRootProperty.cs" />
     <Compile Include="Schema\Extension.cs" />
-    <Compile Include="Schema\Functions.cs" />
+    <Compile Include="Schema\Extras.cs" />
     <Compile Include="Schema\Gltf.cs" />
     <Compile Include="Schema\GltfProperty.cs" />
     <Compile Include="Schema\Image.cs" />
+    <Compile Include="Schema\Indices.cs" />
     <Compile Include="Schema\Material.cs" />
     <Compile Include="Schema\Mesh.cs" />
     <Compile Include="Schema\Node.cs" />
+    <Compile Include="Schema\Normaltextureinfo.cs" />
```

Comment:
Well, VS just added them as what casing they are on disk. But what is writing the files with this casing on disk?